Sources/App/Extensions/EnvironmentProperties.swift

```diff
@@ -59,6 +59,24 @@ extension Environment {
         }
     }
 
+    /// Name of the minimum log level to start logging
+    ///
+    /// Known levels are, in decreasing order of verbosity:
+    ///  - `verbose`
+    ///  - `debug`
+    ///  - `info`
+    ///  - `warning`
+    ///  - `error`
+    ///  - `fatal`
+    /// Any log that is higher that the `minimumLogLevel` in this list will be filtered out. e.g. a `minimumLogLevel` of `info`
+    /// will filter out `verbose` and `debug` logs and will only print `info`,`warning`,`error` and `fatal` logs
+    static func minimumLogLevel() -> LogLevel {
+        guard let value: String = Environment.get("MINIMUM_LOG_LEVEL") else {
+            return .verbose // If env var is not set, log everything and don't filter out any log
```

Comment:
By default should be `info`, we shouldn't force consumers to provide a log level, verbose or even debug is simply too much logging by default.

Reply:
Yeah I quite hesitated on that. My take was: don't break existing behavior. So if you don't set the env var, print the same logs as before we had that filtering feature 🤷‍♂ 

I'm ok to set it to .info though, will push that after my meeting

Reply:
Fixed in 99272de82626b7153aa3a8562e8db3466e5e73c1